### ISSUE TITLE
ERROR: Condition "skeleton->size == 0" is true. Returning: RID()

### ISSUE THREAD

**Issue by enxas** [godotengine/godot#77780]
### Godot version

v4.1.dev4.mono.official [5c2295ff5]

### System information

Godot v4.1.dev4.mono - Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1660 Ti with Max-Q Design (NVIDIA; 31.0.15.3179) - Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 Threads)

### Issue description

When I open my project console gets filled with these errors.

```
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: This function in this node can only be accessed from either the main thread or a thread group. Use call_deferred() instead.
   at: (scene/main/canvas_item.cpp:120)
ERROR: Condition "skeleton->size == 0" is true. Returning: RID()
   at: skeleton_get_3d_uniform_set (./servers/rendering/renderer_rd/storage_rd/mesh_storage.h:693)
ERROR: Condition "skeleton->size == 0" is true. Returning: RID()
   at: skeleton_get_3d_uniform_set (./servers/rendering/renderer_rd/storage_rd/mesh_storage.h:693)
```
### Steps to reproduce (from fresh new project)
Steps to reproduce `ERROR: Condition "skeleton->size == 0" is true. Returning: RID()` error.
1. Create new project.
1.1 Godot version used: v4.1.beta2.mono.official [a2575cba4]).
1.2 I have this setting On: Editor -> Editor Settings -> General -> Interface -> Scene Tabs -> "Restore Scenes On Load".
2. Copy/Paste .glb model with a skeleton to the project folder.
3. Double click .glb file in the editor and click Reimport.
4. Right click .glb file and select New Inherited Scene.
5. Click ctrl+s to save that scene for example player.tscn.
6. Right click player.tscn and select Instantiate.
7. Save everything and restart Godot editor. When launching editor you will see message `ERROR: Condition "skeleton->size == 0" is true. Returning: RID()` in console and Output.

### Steps to reproduce (from MRP)

Open project

### Minimal reproduction project

MRP uses v4.1.beta1.mono.official [828ec2c5d], but same error still persists
[MRP.zip](https://github.com/godotengine/godot/files/11703993/MRP.zip)

**Comment by AThousandShips** [comment]
Please add an MRP, we can't do anything without actual details on what you're doing

**Comment by enxas** [comment]
Sorry, I figured it out. My MeshInstance3D was missing a Skeleton3D reference. When I added it, on project launch the problem went away. I tried removing skeleton reference again and console is printing these error non stop now. I guess this issue can be closed now.
![image](https://github.com/godotengine/godot/assets/8630984/8352edfd-defb-4dcb-ac3d-443478378ce6)
```
ERROR: Condition "!uniform_set" is true.
   at: draw_list_bind_uniform_set (drivers/vulkan/rendering_device_vulkan.cpp:7208)
```
